ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -561,7 +561,6 @@ function write(filter) {
         default:
           throw new Error('Unknown spatial filter type: ' + filter.type)
       }
-      break
```

Comment:
❓ Is this intentional? The result will be extra case checking instead of short-circuit out of the switch. I think we would want to keep the `break`s

Reply:
the linter marks the line as unreachable which is correct. every `if` ends in a return

Reply:
the breaks are unreachable since every case returns